root/components/Aliases/AliasTableBody.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const AliasTableRow = require('./AliasTableRow');
+const {l} = require('../../static/scripts/common/i18n');
+
+type Props = {
+  +aliases: $ReadOnlyArray<AliasT>,
+  +allowEditing: boolean,
+  +entity: $Subtype<EntityT>,
+};
+
+
+const AliasTableBody = ({aliases, allowEditing, entity}: Props) => {
```

Comment:
I originally wanted to write this as {aliases, ...props}
but I don't know how to make flow work with that.

Reply:
Seems to only work if the props for `AliasTableRow` are inexact: https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBjOA7AzgFzACUBTAQw0IF4wAnEgRwFcBLegCgHJ6L9OBKANzp8ATwAOJMAAVaccbgCMYGgG8APqjBgA1GQBcYArRbYA5gBotugEaHjpy9Z0Z7+E+avqAvsNTBgGTkFACYwAFsmAjAbKVMSAA9eMHZsODANMAA6HLAffizUMUkg+VwwtWc7I3dHK20XNw8nX3QsPEIAYThw8RwSbGoU8UNZMpD+FQA+FOtxLJtdMHmMVCE2nGiE0eClFQyDME5OCxjDY9PXI85W9ujVMlOcrPoCb32EvzvCKDh0mgAPN1ev1Bhlnq98O9gFNhEA